### PR TITLE
creating a HL app

### PR DESCRIPTION
**Pull request by sharath9620**
please review and merge this PR.
let me know for any modifications.

this change creates HL app if it does not exist.

### CODE REVIEW

File: src/hlRestCall.py [hunk 1]
```diff
@@ -62,7 +62,8 @@ def get_app_id(self,app_name):
             for id in json:
                 if id['name'].lower()==app_name.lower():
                     return id['id']
-            raise KeyError (f'Highlight application not found: {app_name}')
+            #raise KeyError (f'Highlight application not found: {app_name}')
```
Comment (NevinK) [inline src/hlRestCall.py:65]: why did you remove the exception?

Comment (sharath9620) [reply src/hlRestCall.py:65]: previously if the application is not present in the HL it was showing the error but now it will create the application.
